### PR TITLE
test miniconda rc1

### PR DESCRIPTION
**Pull request by ocefpaf**
Pinging @msarahan for awareness but don't worry about following this PR.
I'll report upstream any issues I find here.

### CODE REVIEW

File: .appveyor.yml [hunk 1]
```diff
@@ -19,7 +19,8 @@ install:
        throw "There are newer queued builds for this pull request, failing early." }
 
   - curl -fsS -o miniconda3.exe https://repo.anaconda.com/pkgs/misc/previews/miniconda/4.6.9/Miniconda3-4.6.9.rc2.unsigned-Windows-x86_64.exe
-  - miniconda3.exe /InstallationType=JustMe /AddToPath=0 /RegisterPython=0 /S /D=%PYTHON%
+  - start /wait "" miniconda3.exe /InstallationType=JustMe /RegisterPython=0 /S /D=%PYTHON%
```
Comment (msarahan) [inline .appveyor.yml]: yeah, if you don't do this, it just goes into the background

**Comment by msarahan** [comment]
Jeez, that's not a helpful solver message.

https://ci.appveyor.com/project/ocefpaf/notebooks-demos/builds/23519492#L84

**Comment by ocefpaf** [comment]
@msarahan the Windows failure was fixed by https://github.com/conda-forge/scikit-learn-feedstock/pull/92, scikit-learn was missing from `conda-forge` and `strict` did the right thing of refusing to install numpy with openblas and scipy with mkl.

**Comment by ocefpaf** [comment]
I reverted this back to the stable URLs and I'm no longer using conda-canary, but both the miniconda rc1 and the conda-canary worked just fine for this environment setup.

**Pull request merged**